src/sqlfluff/rules/L059.py

```diff
@@ -87,6 +89,17 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
         self.prefer_quoted_identifiers: bool
         self.ignore_words: str
         self.ignore_words_regex: str
+        self.force_enable: bool
+        # Some dialects allow quotes as PART OF the column name. In other words,
+        # these are two different columns:
+        # - date
+        # - "date"
+        # For safety, disable this rule by default in those dialects.
```

Comment:
Can we add a `note` to docstring to explain this?

similar to this one: https://docs.sqlfluff.com/en/stable/rules.html#sqlfluff.core.rules.Rule_L026

Reply:
Done!